cross/libdrm/PLIST

```diff
@@ -0,0 +1,83 @@
+include/libdrm/amdgpu_drm.h
+include/libdrm/amdgpu.h
+include/libdrm/drm_fourcc.h
+include/libdrm/drm.h
+include/libdrm/drm_mode.h
+include/libdrm/drm_sarea.h
+include/libdrm/i915_drm.h
+include/libdrm/i915_perfmon.h
+include/libdrm/intel_aub.h
+include/libdrm/intel_bufmgr.h
+include/libdrm/intel_debug.h
+include/libdrm/intel_mocs_table.h
+include/libdrm/intel_perfmon.h
+include/libdrm/mach64_drm.h
+include/libdrm/mga_drm.h
+include/libdrm/nouveau_drm.h
+include/libdrm/nouveau/nouveau.h
+include/libdrm/nouveau/nvif/cl0080.h
+include/libdrm/nouveau/nvif/cl9097.h
+include/libdrm/nouveau/nvif/class.h
+include/libdrm/nouveau/nvif/if0002.h
+include/libdrm/nouveau/nvif/if0003.h
+include/libdrm/nouveau/nvif/ioctl.h
+include/libdrm/nouveau/nvif/unpack.h
+include/libdrm/qxl_drm.h
+include/libdrm/r128_drm.h
+include/libdrm/r600_pci_ids.h
+include/libdrm/radeon_bo_gem.h
+include/libdrm/radeon_bo.h
+include/libdrm/radeon_bo_int.h
+include/libdrm/radeon_cs_gem.h
+include/libdrm/radeon_cs.h
+include/libdrm/radeon_cs_int.h
+include/libdrm/radeon_drm.h
+include/libdrm/radeon_surface.h
+include/libdrm/savage_drm.h
+include/libdrm/sis_drm.h
+include/libdrm/tegra_drm.h
+include/libdrm/vc4_drm.h
+include/libdrm/via_drm.h
+include/libdrm/vmwgfx_drm.h
+include/libkms/libkms.h
+include/xf86drm.h
+include/xf86drmMode.h
+lib/libdrm_amdgpu.la
+lib/libdrm_amdgpu.so
+lib/libdrm_amdgpu.so.1
+lib/libdrm_amdgpu.so.1.0.0
+lib/libdrm_intel.la
+lib/libdrm_intel.so
+lib/libdrm_intel.so.1
+lib/libdrm_intel.so.1.0.0
+lib/libdrm.la
+lib/libdrm_nouveau.la
+lib/libdrm_nouveau.so
+lib/libdrm_nouveau.so.2
+lib/libdrm_nouveau.so.2.0.0
+lib/libdrm_radeon.la
+lib/libdrm_radeon.so
+lib/libdrm_radeon.so.1
+lib/libdrm_radeon.so.1.0.1
+lib/libdrm.so
+lib/libdrm.so.2
+lib/libdrm.so.2.4.0
+lib/libkms.la
+lib/libkms.so
+lib/libkms.so.1
+lib/libkms.so.1.0.0
+lib/pkgconfig/libdrm_amdgpu.pc
+lib/pkgconfig/libdrm_intel.pc
+lib/pkgconfig/libdrm_nouveau.pc
+lib/pkgconfig/libdrm.pc
+lib/pkgconfig/libdrm_radeon.pc
+lib/pkgconfig/libkms.pc
+share/man/man3/drmAvailable.3
+share/man/man3/drmHandleEvent.3
+share/man/man3/drmModeGetResources.3
+share/man/man7/drm.7
+share/man/man7/drm-gem.7
+share/man/man7/drm-kms.7
+share/man/man7/drm-memory.7
+share/man/man7/drm-mm.7
+share/man/man7/drm-ttm.7
```

Comment:
Please remove man pages and other stuff not needed and prefix the rest according to https://github.com/SynoCommunity/spksrc/wiki/PLIST-files#example